mrtt-ui/src/components/LoadingIndicatorOverlay.js

```diff
@@ -0,0 +1,20 @@
+import { Backdrop, CircularProgress } from '@mui/material'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import { LoadingWrapper } from '../styles/containers'
+import theme from '../styles/theme'
+
+const LoadingIndicatorOverlay = ({ isVisible }) => {
+  return (
+    <Backdrop sx={{ zIndex: (theme) => theme.zIndex.drawer + 1 }} open={isVisible}>
```

Comment:
Is there a reason `sx` is being used instead of styled components? Is there a convention in the codebase?

Reply:
In an ideal world, I avoid inline styles, but this project is super budget, so we are going to quick and cheap where reasonable. Normally I would bust out a styled component to avoid this for sure.